packages/providers/inpage-providers-hub/src/connectButtonHack/universal/config.ts

```
@@ -351,14 +357,14 @@
           ...basicWalletInfo['metamask'],
           container: '#metamask',
           afterUpdate(textNode, img) {
-            textNode.parentElement && (textNode.parentElement.style.textAlign = 'left');
+            textNode?.parentElement && (textNode.parentElement.style.textAlign = 'left');
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)